src/commonTest/kotlin/expressions/MissingSomeTest.kt

```diff
@@ -0,0 +1,68 @@
+package expressions
+
+import JsonLogicEngine
+import TestInput
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+
+class MissingSomeTest : FunSpec({
```

Comment:
Is it possible to add more descriptive test case names? for now it does not very readable
![Screenshot 2022-01-28 at 11 32 24](https://user-images.githubusercontent.com/82953872/151531749-62254382-0548-445e-aa9f-c7b471dbe479.png)

Reply:
👍🏽 Sure

After:
![image](https://user-images.githubusercontent.com/84186160/151751213-91d9ab1e-7772-4a7b-b9d6-0f4591944715.png)